test/elements/useElement.test.ts

```diff
@@ -18,9 +18,9 @@ describe('useElement', () => {
   let update: jest.Mock;
 
   beforeEach(() => {
-    mount = jest.fn();
+    mount = jest.fn(() => Promise.resolve());
```

Comment:
```suggestion
    mount = jest.fn().mockResolvedValue();
```
nbd if you don't take these

Reply:
I was back and forth on this because it requires a value.

Reply:
Accepted tho